test/fixtures/invalid_certificate_2.json

```diff
@@ -112,59 +112,59 @@
         },
         "ChemicalComposition": {
           "C71": {
-            "Actual": 101,
+            "Actual": "101",
```

Comment:
It should be less then 100 as it is given in percent.

Reply:
Fixed 👍🏼

Reply:
since it's an invalid fixture isn't it the purpose to check that a value above "100" won't pass the pattern validation ?

Reply:
@stiebitzhofer

Reply:
Changed back to 101.